cob_gazebo_worlds/launch/world.launch

```diff
@@ -16,10 +16,13 @@
 	<env name="GAZEBO_MODEL_PATH" value="$(arg pkg_gazebo_worlds)/Media/models:$(optenv GAZEBO_MODEL_PATH)"/>
 
 	<!-- upload default configuration parameters -->
-	<include file="$(arg pkg_env_config)/upload_navigation_goals.launch">
+	<include file="$(find cob_default_env_config)/upload_navigation_goals.launch">
+		<arg name="pkg_env_config" value="$(arg pkg_env_config)"/>
```

Comment:
well, it adds an argument to a lot of files without being used there...

maybe we should call the `upload_navigation_goal.launch` and `upload_object_locations.launch` directly from `cob_bringup_sim/launch/robot.launch`...
this way, we could actually get rid of the dependency to `cob_default_env_config` within `cob_gazebo_worlds`